mobify/platform-scaffold
# Typography update to product tile, semi-bold and bold adjusted

(brief description / PR title)

https://mobify.atlassian.net/browse/WEB-1332

## Changes
- updated PLP product tile to use system font
- updated cart product tile to use system font
- updated added to cart modal product tile to use system font
- changed semi-bold weight to 500 to match mock
- changed price class in all instances to adopt bold weight
- removed extralight from PLP h1 weith

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- browse to plp to view product tile
- add item to cart to view product tile in modal
- view cart to see product tile in cart
